commands/helper_completion.go

```diff
@@ -0,0 +1,49 @@
+package commands
+
```

Comment:
I wasn't sure about the filename, since most files inside this package should be commands, but at least `env.go` isn't.
I didn't find a good pattern, this was just what I came up with.

Reply:
LGTM